drivers/sensor/bme680/bme680.c

```diff
@@ -414,6 +414,8 @@ static int bme680_init(const struct device *dev)
 
 	data->i2c_slave_addr = DT_INST_REG_ADDR(0);
 
+	k_msleep(2); /* Start up delay */
```

Comment:
just wondering is there is no better way to handle such delays
it will delay application start which in some cases may be critical issue

@anangl is there any async model for drivers bringup now? when we done motion sensors I found none such thing soin the end we return ebusy from api calls until driver is up (application tracks that).